build/Dockerfile

```diff
@@ -1,5 +1,6 @@
 # syntax=docker/dockerfile:1.2
 ARG BUILD_OS=debian
+ARG NGINX_PLUS_VERSION=r24
```

Comment:
we should document the behavior here https://docs.nginx.com/nginx-ingress-controller/installation/building-ingress-controller-image/  : when  building an NGINX Plus image, the Dockerfile will install the latest patch version of the major NGINX Plus version supported in by the Ingress Controller release.

Reply:
Thinking about this again, we never mentioned before what version of NGINX would be installed. Not sure if we should start now.

Reply:
previously, the version was fixed. Not it is floating, which means at different points of time, the `docker build` command can produce an image with different versions. I think we should mention that. This is a change from previous behavior.

Reply:
Because we will not release a new version of KIC on every patch release, we should explicitly mention that. Something like this:
After a [new NGINX Plus patch version](https://docs.nginx.com/nginx/releases/) is released, make sure to rebuild the image so that you always running the latest version.